package.json

```diff
@@ -6,6 +6,7 @@
         "node": ">=22.0.0"
     },
     "scripts": {
+        "prepare": "yarn build",
         "prepack": "yarn build",
```

Comment:
Is prepack still needed? I'd imagine not

https://docs.npmjs.com/cli/v9/using-npm/scripts#life-cycle-scripts